Toolset/home.livecodescript

```diff
@@ -723,12 +672,359 @@ private command revInternal__SetupSplash
+   put "false" into tPropsArray["cSelectGrouped"]
+   put "false" into tPropsArray["cSEHideMessageBox"]
+   put "" into tPropsArray["cREVAnimationTopLeft"]
+   put "1,2,3,4,5" into tPropsArray["cVideoclipTabs"]
+   put "true" into tPropsArray["cREVMacOSMenubar"]
+   put "" into tPropsArray["cREVOpenTabPositions"]
+   put "false" into tPropsArray["cREVProfileReadOnly"]
+   put "Yellow" into tPropsArray["revHelpHiliteColor"]
+   put "true" into tPropsArray["cToolbarText"]
+   put "Pink" into tPropsArray["Pink"]
+   put "true" into tPropsArray["cREVScriptDebugMode"]
+   put "21" into tPropsArray["cLabelFieldHeight"]
+   put "120" into tPropsArray["cREVGraphicWidth"]
+   put "417,99" into tPropsArray["cREVMsgTabRectMultiple Lines"]
+   put "30" into tPropsArray["Num Recent Files"]
+   put "568,259" into tPropsArray["cREVProfileEditorTopLeft"]
+   put "417,113" into tPropsArray["cREVMsgTabRectfrontScripts"]
+   put "1088781994" into tPropsArray["cTipsShown"]
+   put "11" into tPropsArray["cREVRotatePrev3"]
+   put "" into tPropsArray["cREVScriptSize"]
+   put "338" into tPropsArray["cREVButtonSecondTabHeight"]
+   put "true" into tPropsArray["cREVHelpText"]
+   put "160" into tPropsArray["cTabButtonHeight"]
+   put "287,261" into tPropsArray["cREVMsgTabRectrevstack"]
+   put "2" into tPropsArray["cShowTips"]
+   put "false" into tPropsArray["cAppOverview"]
+   put "" into tPropsArray["cCustomerNumber"]
+   put "417,113" into tPropsArray["cREVMsgTabRectGlobal Properties"]
+   put "true" into tPropsArray["cShowAudioclipclips"]
+   put "revolutionli" into tPropsArray["cREVDBName"]
+   put "true" into tPropsArray["cDocsQuickLinksVisible"]
+   put "true" into tPropsArray["cStartPageShow"]
+   put "278" into tPropsArray["cREVErrorsPaletteExpandedHeight"]
+   put "" into tPropsArray["cREVScriptColor"]
+   put "true" into tPropsArray["cShowAudioclips"]
+   put "filtering" into tPropsArray["revHelpListMethod"]
+   set the customProperties[""] of stack "revPreferences" to tPropsArray
+   delete variable tPropsArray
+   
+   -- cREVMessageBoxIndex
+   put "" into tPropsArray["Single Line"]
+   set the customProperties["cREVMessageBoxIndex"] of stack "revPreferences" to tPropsArray
+   delete variable tPropsArray
+   
+   -- cREVMessageBoxNoMessages
+   put "" into tPropsArray["Single Line"]
+   set the customProperties["cREVMessageBoxNoMessages"] of stack "revPreferences" to tPropsArray
+   delete variable tPropsArray
+   
+   -- revHelpWindowPosition
+   set the customProperties["revHelpWindowPosition"] of stack "revPreferences" to tPropsArray
```

Comment:
must be a set with no keys as I had a breakpoint on arrays in my script because I wasn't sure if I needed to deal with them